lib/rubocop/cop/i18n/gettext/decorate_function_message.rb

```diff
@@ -7,16 +7,14 @@ class DecorateFunctionMessage < Cop
           def on_send(node)
             method_name = node.loc.selector.source
             return if !/raise|fail/.match(method_name)
-            if method_name == "raise"
-              receiver_node, method_name, *arg_nodes = *node
+            if method_name == "raise" || method_name == "fail"
+              _, method_name, *arg_nodes = *node
               if !arg_nodes.empty? && arg_nodes[0].type == :const && arg_nodes[1]
-                how_bad_is_it(node, method_name, arg_nodes[1])
-              end
-            elsif method_name == "fail"
-              receiver_node, method_name, *arg_nodes = *node
-              if !arg_nodes.empty?
-                how_bad_is_it(node, method_name, arg_nodes[0])
+                arg_node = arg_nodes[1]
+              else
+                arg_node = arg_nodes[0]
```

Comment:
Ugh, nevermind this causes the cop to look at all kinds of unintended things. Going to play around with this more.